pkg/sql/lex/all_keywords.go

```diff
@@ -9,19 +9,6 @@
 // licenses/APL.txt.
 
 // +build all-keywords
```

Comment:
this actually doesn't work :( 

https://github.com/bazelbuild/rules_go/issues/2750#issuecomment-740188977

we'll need to either move this `all_keywords` thing to a new package, or try the starlark rule idea that was suggested

Otherwise it is just a package comment.